app/src/main/res/values-de/strings.xml

```diff
@@ -667,4 +667,5 @@
   <string name="no_location_found_message">Wie wäre es, den Ort hinzuzufügen, an dem das Bild aufgenommen wurde?\nOrtsangaben helfen Wiki-Benutzern, dein Bild zu finden, und machen es so viel nützlicher.\nDanke schön!</string>
   <string name="add_location">Standort hinzufügen</string>
   <string name="feedback_sharing_data_alert">Bitte entferne aus dieser E-Mail alle Informationen, die du nicht öffentlich mitteilen möchtest. Bitte beachte auch, dass deine E-Mail-Adresse, mit der du schreibst, sowie der dazugehörige Name und das Profilbild öffentlich sichtbar sind.</string>
+  <string name="copyright_popup">Bitte lade nur Bilder hoch, die du selbst aufgenommen hast. Wer urheberrechtlich geschützte Bilder hochlädt, wird gesperrt. Dies gilt auch für die Beta-Variante. Vielen Dank, dass du die App getestet hast!</string>
```

Comment:
I guess these changes are unrelated?
Would you mind rebasing from master?
If nothing else is working, you can also create a new branch from master, apply the right changes, and send a new pull request :-)

Reply:
@nicolas-raoul those changes are from some else's PR. I actually by mistake changed the author that's why it is showing as if i have made those changes. But actually I merged the master branch and changed the author probably that's why it is showing in my commit.

Reply:
Would you mind creating from scratch a new branch and a new pull request?